cmd/kill/killer/kill_unix.go

```diff
@@ -0,0 +1,29 @@
+// +build linux darwin dragonfly freebsd netbsd openbsd
+
+package killer
+
+import (
+	"syscall"
+)
+
+func Kill(pids []int, safe bool) (map[int]error) {
+	var errm = make(map[int]error)
+	var err error
+
+	for _, pid := range pids {
+		err = syscall.Kill(pid, syscall.SIGTERM)
+		if err != nil {
+			if safe == true {
+				goto errloop
+			}
+			err = syscall.Kill(pid, syscall.SIGKILL)
+			if err != nil {
+				goto errloop
+			}
+		}
+		continue
```

Comment:
I like the idea of one single exit point for errors too. That guarantees the error is handled equally and that cleanup (close file descs, free memory, etc) is done once and only once. That's good.
But inside loops, I don't know it's always good.

If the loop is small, as in this case with only two error points, it's pretty clear the logic. Maybe makes sense in more complicated loops, maybe with 4 or 5 error points that must be handled in the same way.

I have the feelings that this `for` isn't a `for` (as someone would expect) and this could be more dangerous than the error being handled multiple times.

@katcipis what do you think?